pxtblocks/fields/field_position.ts

```diff
@@ -70,6 +71,9 @@ namespace pxtblockly {
             const simFrame = this.getSimFrame();
             if (!simFrame) return;
 
+            if (FieldPosition.eyedropperActivated_) return;
+            FieldPosition.eyedropperActivated_ = true;
```

Comment:
Could we query the DOM If a particular element is present?

Reply:
Yes. this.selectorDiv_